FAQ.md

```diff
@@ -6,27 +6,37 @@ _The first rule of FAQ is don't bikeshed the FAQ, leave that for
 Please contribute to the FAQ! Found an explanation in an issue or pull request helpful?
 Summarize the question and quote the reply, linking back to the original comment.
 
+## Concepts
 * [Does `dep` replace `go get`?](#does-dep-replace-go-get)
 * [Why is it `dep ensure` instead of `dep install`?](#why-is-it-dep-ensure-instead-of-dep-install)
+* [What is a direct or transitive dependency?](#what-is-a-direct-or-transitive-dependency)
+
+## Configuration
 * [What is the difference between Gopkg.toml (the "manifest") and Gopkg.lock (the "lock")?](#what-is-the-difference-between-gopkgtoml-the-manifest-and-gopkglock-the-lock)
```

Comment:
I don't have a definitive opinion but I feel this one belongs under Concepts since the answer. The question could be updated to be:

> What is the difference between the "manifest" (`Gopkg.toml`) and the "lock" (`Gopkg.lock`)?